modules/ROOT/pages/save-builtin-dashboard-as-custom.adoc

```diff
@@ -0,0 +1,25 @@
+= Save a Built-in Dashboard as a Custom Dashboard
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+To simplify the process of creating a custom dashboard from scratch, you can save any of the built-in dashboards that are included in the Anypoint Monitoring tabs as a base template for a custom dashboard that you can then customize to suit your needs. This allows you to take advantage of the flexibility and features offered in custom dashboards, such as troubleshooting, setting up alerts, or comparing multiple resources within one dashboard. 
+
+Follow these steps to save a built-in dashboard as a custom dashboard. 
+
+. In Anypoint Monitoring, click *Built-in dashboards* in the menu on the left.
```

Comment:
bold for AM? Include first step from Anypoint Platform for first step in a new topic?